casper/src/main/resources/PoS.rhox

```diff
@@ -486,7 +620,7 @@ in {
                         @RevVault!("unforgeableAuthKey", unf, *unfAuthKeyCh) |
                         for (@authKey <- unfAuthKeyCh) {
                           // Transfer funds from validator vault to the PoS vault.
-                          @vault!("transfer", posRevAddress, balance, authKey, *transferDoneCh) |
+                          @vault!("transfer", posVaultAddr, balance, authKey, *transferDoneCh) |
```

Comment:
_moveFunds_ is not used anymore. I missed that when removing per validator vaults.